server/lib/tuist/storage.ex

```diff
@@ -281,15 +315,19 @@ defmodule Tuist.Storage do
   end
 
   defp region_headers(actor) do
-    case actor do
-      %Account{region: :europe} ->
-        [{"X-Tigris-Regions", "eur"}]
+    if has_custom_storage?(actor) do
```

Comment:
note that this setting in the dashboard will be confusing – I think it's fine as it's a one-off, but something to keep in mind probably.